### ISSUE TITLE
How to colocate persistence rules with reducer

### ISSUE THREAD

**Issue by thenewt15** [rt2zz/redux-persist#586]
There's a bullet point in the readme, specifying reasons for v5. It says,
> ability to colocate persistence rules with the reducer it pertains to

What does this mean? Are there any examples of how this works?

I'm aware of transforms as a way to customize the state that is persisted, but I didn't think transforms were something to be colocated with your reducers. This must refer to something else...

**Comment by rt2zz** [comment]
This is not well documented atm, but in short `persistReducer` can be wrapped around any reducer. Meaning say you have a `content` reducer, you can actually wrap the reducer with persistReducer before exporting it:
```js
// reducers/content.js
function content(state, action) {
  // ...
}

const persistConfig = {
  key: 'content',
  // ...
}

export default persistReducer(persistConfig, content)

**Comment by thenewt15** [comment]
Wow, this is a great feature and provides a solution to Issues [134](https://github.com/rt2zz/redux-persist/issues/134) and [266](https://github.com/rt2zz/redux-persist/issues/266).

I'd like to elaborate on this for others...

Let's say your state object has an `auth` key and that you want to persist `auth.currentUser` but NOT `auth.isLoggingIn`. 

What you can do is wrap your AuthReducer with the redux-persist PersistReducer, and then blacklist the `isLoggingIn` key.

```
// AuthReducer.js
import storage from 'redux-persist/lib/storage';
import { persistReducer } from 'redux-persist';

const INITIAL_STATE = {
  currentUser: null,
  isLoggingIn: false,
};

const AuthReducer = (state = INITIAL_STATE, action) => {
  // reducer implementation
};

const persistConfig = {
  key: 'auth',
  storage: storage,
  blacklist: ['isLoggingIn']
};

export default persistReducer(persistConfig, AuthReducer);
```

**Comment by rt2zz** [comment]
@thenewt15 exactly! thanks for PRing the readme updates - definitely want to get this more visible
